features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/DataSyncStatusService.kt

```diff
@@ -19,11 +19,8 @@ package org.dash.wallet.features.exploredash.repository
 
 import kotlinx.coroutines.flow.Flow
 import org.dash.wallet.common.data.Resource
-import java.lang.Exception
 
 interface DataSyncStatusService {
-    suspend fun setSyncProgress(progress: Double)
-    suspend fun setSyncError(exception: Exception)
```

Comment:
Interface users don't actually need access to these methods since they aren't supposed to change the state. The implementation can be injected into the worker which needs full access.